bootstrap/sponge/src/main/java/org/geysermc/platform/sponge/GeyserSpongeConfiguration.java

```diff
@@ -41,6 +43,13 @@
     private File dataFolder;
     private ConfigurationNode node;
 
+    /**
+     * If the config was originally 'auto' before the values changed
+     */
+    @Getter
+    @Setter
```

Comment:
I would likely remove these to follow the rest of the config, unless that causes issues.